distribution/docs/src/main/resources/_contents/_running/monitoring-contents.adoc

```diff
@@ -0,0 +1,140 @@
+
+The ${branding} contains many tools to monitor system functionality, usage, and overall system health.
+
+==== Metrics Reporting
+
+Metrics are available in several formats and levels of detail.
+
+Complete the following procedure now that several queries have been executed.
+
+. Select *${branding}-Platform*
+. Select *Metrics* tab
+. For individual metrics, choose the format desired from the desired timeframe column:
+.. PNG
+.. CSV
+.. XLS
+. For a detailed report of all metrics, at the bottom of the page are selectors to choose time frame and summary level.
+A report is generated in _xls_ format.
+
+==== Managing Logging
+
+The ${branding} supports a dynamic and customizable logging system including log level, log format, log output destinations, roll over, etc.
+
+===== Configuring Logging
+
+Edit the configuration file `[${branding-lowercase}_install_dir]/etc/org.ops4j.pax.logging.cfg]`
+
+===== ${branding} log file
+
+The name and location of the log file can be changed with the following setting:
+
+`log4j.appender.out.file=<KARAF.DATA>/log/${branding-lowercase}.log`
+
+===== Controlling log level
+
+A useful way to debug and detect issues is to change the log level:
+
+`log4j.rootLogger=DEBUG, out, osgi:VmLogAppender`
+
+===== Controlling the size of the log file
+
+Set the maximum size of the log file before it is rolled over by editing the value of this setting:
+
+`log4j.appender.out.maxFileSize=20MB`
+
+===== Number of backup log files to keep
+
+Adjust the number of backup files to keep by editing the value of this setting:
+
+`log4j.appender.out.maxBackupIndex=10`
+
+===== Enabling logging of inbound and outbound SOAP messages for the ${branding} SOAP endpoints
+
+By default, the ${branding} start scripts include a system property enabling logging of inbound and outbound SOAP messages.
+
+`-Dcom.sun.xml.ws.transport.http.HttpAdapter.dump=true`
+
+In order to see the messages in the log, one must set the logging level for `org.apache.cxf.services` to `INFO`. By default, the logging level for `org.apache.cxf` is set to `WARN`.
+
+`${branding-lowercase}${at-symbol}local>log:set INFO org.apache.cxf.services`
+
+===== Logging External Resources
+
+Other appenders can be selected and configured.
+
+For more detail on configuring the log file and what is logged to the console a handy reference is `http://karaf.apache.org/manual/latest-2.2.x/users-guide/logging-system.html`
```

Comment:
The current link does not work. I think we can use `http://karaf.apache.org/manual/latest/#_log` pretty safely.